components/nodemanager-service/tests/mgrtesthelpers/mgr-test-helpers.go

```diff
@@ -101,6 +105,26 @@ func AddAzureManager(ctx context.Context, mgrClient manager.NodeManagerServiceCl
 	return mgrClient.Create(ctx, &azureMgr)
 }
 
+func AddGCPManager(ctx context.Context, mgrClient manager.NodeManagerServiceClient, mgrType string) (*manager.Ids, error) {
+	// decode our encoded key
+	decoded, err := base64.StdEncoding.DecodeString(os.Getenv("GOOGLE_CREDS_JSON"))
+	if err != nil {
+		return nil, err
+	}
+	escaped := strings.ReplaceAll(string(decoded), "\n", "\\n")
```

Comment:
Will other special characters cause problems here, like `\t`? I guess since we control that input we don't have to worry so much, I'm just curious if we could prevent a source of bugs by relying on a more general purpose encoding/escaping utility.

Reply:
I just noticed this is in a test helper function. Probably not worrying about that ^^ 😅